src/Files.App/ViewModels/UserControls/Widgets/RecentFilesWidgetViewModel.cs

```diff
@@ -232,6 +234,22 @@ public void NavigateToPath(string path)
 			catch (Exception) { }
 		}
 
+		public bool CheckIsRecentItemsEnabled()
```

Comment:
Would this make more sense in the service?

Reply:
I don't think so at this moment.
I can't think of other places that use this method. I had an idea to create WindowsConfigurationsService but I'm not sure...